doc/wg/core/notes/core-notes-2025-01-22.md

```diff
@@ -0,0 +1,135 @@
+# Tock Meeting Notes 2025-01-22
+
+---
+
+Agenda
+- Updates
+- WG Report ups (Network WG)
+- TRD104: Explicit type summary for syscalls
+- CapabilityPtr vs. MachinePtr
+
+---
+
+## Attendees
+ - Branden Ghena
+ - Amit Levy
+ - Leon Schuermann
+ - Pat Pannuto
+ - Johnathan Van Why
+ - Hudson Ayers
+ - Lawrence Esswood
+ - Brad Campbell
+ - Kat Fox
+ - Alexandru Radovici
+ - Tyler Potyondy
+ - Chris Frantz
+
+## Updates
+
+ - (full agenda; tight meeting; nothing substantial)
+ - Branden: Scratch network WG update; bi-weekly meetings and this was off week.
+
+## TRD104: Explicit type summary for syscalls ( https://github.com/tock/tock/pull/4228 )
+
+### Agenda Notes
+
+A proposal to add consistent and explicit type information to syscall definitions in TRD104.
+
+#### Context
+
+https://github.com/tock/tock/pull/4228
+
+#### Decisions
+
+1. Do we to include explicit type information for each system call in TRD104? (up or down)
+
+2. If yes, does PR #4228 have exactly the right types? If not, what would needs to change? If unclear, what questions to be resolved?
+
+#### Actions
+
+- Close PR if we (1) is no.
+
+- Merge PR if (2) is yes
+
+### Meeting Notes
+
+ - Pat: History...
+ - Amit: Essentially what the PR does is add a type column for tables in TRD104
+   which document system calls.
+ - Amit: The types currently are the basic integers, etc. Could in the future
+ - Johnathan: The motivation is pointers versus integers, which there's agreement about.
+ - Johnathan: The bear it pokes, without agreement, is pointers versus `usize`, which gets messier on 64-bit, where I don't think we have agreement yet.
```

Comment:
The bear it pokes, unintentionally, is `u32` versus `usize`, which matters on 64-bit, and I don't think we have agreement yet.